app/webserver.py

```diff
@@ -0,0 +1,22 @@
+import random
```

Comment:
Would it be better to document where you got this from? https://docs.python.org/3/library/wsgiref.html#examples